### ISSUE TITLE
Pods installation error

### ISSUE THREAD

**Issue by kostiakoval** [QueryKit/QueryKit#45]
I add QueryKit as a pod to project 
`pod 'QueryKit', '~> 0.11'`

When I build it I get an error
Pods/QueryKit/QueryKit/QueryKit.h:10:9: Include of non-modular header inside framework module 'QueryKit.QueryKit'

![screen shot 2015-11-03 at 11 12 13](https://cloud.githubusercontent.com/assets/1086260/10905634/c95bb1e0-821b-11e5-92e4-dd4132d43a10.png)

**Comment by kostiakoval** [comment]
The solution that works for me is to exclude `QueryKit.h` from a pod.
CocoaPods automatically generated umbrella header

**Comment by wodetian** [comment]
Hello, I'm trying to recreate the error 
This is my ` Podfile` code 

````
source 'https://github.com/CocoaPods/Specs.git'
use_frameworks!
platform :ios, '8.0'

pod 'QueryKit'               # CoreData
```

I try to run, and did not appear this mistake. 

This is my AppDelegate Code
````
//
//  AppDelegate.swift
//  asx
//
//  Created by Mister on 15/11/3.
//  Copyright © 2015年 Mister. All rights reserved.
//

import UIKit
import QueryKit

@UIApplicationMain
class AppDelegate: UIResponder, UIApplicationDelegate {

    var window: UIWindow?


    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
        // Override point for customization after application launch.
        
        print(QueryKitVersionNumber)
        
        return true
    }

    func applicationWillResignActive(application: UIApplication) {
        // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
        // Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
    }

    func applicationDidEnterBackground(application: UIApplication) {
        // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
        // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
    }

    func applicationWillEnterForeground(application: UIApplication) {
        // Called as part of the transition from the background to the inactive state; here you can undo many of the changes made on entering the background.
    }

    func applicationDidBecomeActive(application: UIApplication) {
        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
    }

    func applicationWillTerminate(application: UIApplication) {
        // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
    }


}


```


![a14ec689-a10c-432c-9a79-e5d8f80259cc](https://cloud.githubusercontent.com/assets/6121524/10906140/135114fa-825a-11e5-9378-b9ce1e5e2e25.png)


You can try again  `pod install` Or `pod update`

I wish you success 🍺🍺🍺🍺

**Comment by kostiakoval** [comment]
what cocoapods version are you using?

**Comment by wodetian** [comment]
`0.38.2`

**Comment by kostiakoval** [comment]
here is mine `pod` file 

```
source 'https://github.com/CocoaPods/Specs.git'

platform :ios, '8.0'
inhibit_all_warnings!
use_frameworks!

pod 'QueryKit', '~> 0.11'
```
Same error for empty test project.

**Comment by kostiakoval** [comment]
I'm on 0.39.0

**Comment by wodetian** [comment]
All right. I go to upgrade the `pod` to try

**Comment by kostiakoval** [comment]
here is steps: 
- create simple iOS project
- add QueryKit
- run pod install

Now see the Error

**Comment by wodetian** [comment]
Ok, it is really strange... I don't know how you appear this problem...

![232629e3-2cce-48a5-a9cb-57c1ad36a1aa](https://cloud.githubusercontent.com/assets/6121524/10906339/af43fa20-825b-11e5-831f-c33dfe6c3258.png)
My version is now `0.39.0` 

You try to run my Podfile code 
````
source 'https://github.com/CocoaPods/Specs.git'
use_frameworks!
platform :ios, '8.0'

pod 'QueryKit'               # CoreData
```

**Comment by kostiakoval** [comment]
really weird :( 
I do the same and get this error
```
source 'https://github.com/CocoaPods/Specs.git'
use_frameworks!
platform :ios, '8.0'

pod 'QueryKit'               # CoreData
```

Thanks for help and conversation anyway :) :+1: :+1: 

![screen shot 2015-11-03 at 12 03 10](https://cloud.githubusercontent.com/assets/1086260/10906594/0182852e-8223-11e5-94e8-5217f891056d.png)

**Comment by wodetian** [comment]
Well, I hope you can solve ~

**Comment by wodetian** [comment]
Hello. Today I also met this situation, I found some mistakes............ About the answer in  #46

**Comment by kylef** [comment]
I've released QueryKit 0.12.0 which solves this problem. Thanks for reporting!

**Comment by kostiakoval** [comment]
Great present to end the week. Thanks! :100: :+1:
